sysfs/net_class_test.go

```diff
@@ -29,35 +29,56 @@ func TestNewNetClass(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	var (
+		AddrAssignType   int64 = 3
```

Comment:
Very minor nit: all variables in the scope of a function should start with a lowercase letter.

Reply:
fixed. thanks!